src/components/UI/ChainSelect/ChainSelect.module.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../../index';
+
+.chainSelect {
+  margin: 3px 16px;
+}
+
+.selectedIcon {
+  position: absolute;
+  right: 8px;
+}
```

Comment:
```suggestion
.chainSelect {
  margin: 3px 16px;

  .selectedIcon {
    position: absolute;
    right: 8px;
  }
}
```

Reply:
the select items are outside the select itself in the dom tree, so it doesn't work. I would prefer to it also